WooCommerce/Classes/POS/Utils/Font+WooCommercePOS.swift

```diff
@@ -6,4 +6,7 @@ extension Font {
     static let posDetail: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 16))
     static let posModalBody: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 24))
     static let posModalTitle: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 36), weight: .bold)
+    static let posHeaderTitle: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 40),
```

Comment:
When inspecting designs (TfaZ4LUkEwEGrxfnEFzvJj-fi-2530_16232), it's the same font size as other titles.

Reply:
Thanks for checking! I haven't double checked if the font size has changed since previously implemented, must have been edited since then 🤔 Updated on https://github.com/woocommerce/woocommerce-ios/pull/13585/commits/dc4a70f4d4a05755e41ab68844560161a4381eb6 . I kept it as a separate variable from the modal title despite using the same size.